content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
+created: 2023-11-01
+authors:
   - rufuspollock
   - jefferymartin
+description: Fundamental Wellbeing is the the experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. This page is an introduction to the concept.
+image: /assets/images/fundamental-wellbeing-dr-jeffery-martin.jpg
 ---
-## What is Fundamental Wellbeing
+ At Life Itself, we believe that cultivating being is one of the core tenants of creating a wiser, weller world. Greater spiritual enlightenment, leads to better wellbeing, greater compassion, and honed abilities to be connected to ourselves and others.
+ 
+ While there are many terms for spiritual enlightenment, here we refer to the term Fundamental Wellbeing. In fact, this page is dedicated to Dr. Jeffery Martin's work, the man who coined the term and has spent over a decade researching how people reach Fundamental Wellbeing. 
```

Comment:
Here's an alternate intro i rapidly generated 😉

Dr. Jeffery Martin is a researcher known for his work on Fundamental Wellbeing, which focuses on understanding and cultivating higher states of consciousness and well-being in individuals. He has conducted extensive studies exploring various altered states of consciousness, such as enlightenment, mystical experiences, and non-symbolic consciousness.

Dr. Martin's research aims to identify common patterns and traits associated with these states, helping individuals achieve a more profound sense of well-being and fulfillment. He has developed specific training programs and techniques to guide people into these states.

By studying individuals who have experienced Fundamental Wellbeing, Dr. Martin seeks to uncover the underlying neurological, psychological, and physiological mechanisms that contribute to these states. His work provides valuable insights into human consciousness and offers practical approaches for people to enhance their overall well-being and spiritual experiences.

Reply:
As a bonus here we would probably want to pull out the key points which are:

- Definition of Fundamental Wellbeing (a nice secular term covering "waking up")
- "Maps": Map of Fundamental Wellbeing (the locations and the Martin Matrix). Super useful ...
- "Rafts": the finders protocol and the general ideas behind that of searching for techniques that work, that techniques need to match with you etc.

Reply:
Very good! Since you have been in all the interviews with him and have studied his work more extensively perhaps it's easier for you to write this up fast rather than delegating it to someone who needs to learn from scratch 😉 

I've used your intro and added a this page will cover section. I do not know how to add a ToC which links to headings which would be super useful across the website